command/agent_test.go

```diff
@@ -301,6 +305,7 @@ func TestAgent_HostBasedIDs(t *testing.T) {
 		cmd := &AgentCommand{
 			args: []string{
 				"-data-dir", tmpDir,
+				"-bind", "127.0.0.1",
```

Comment:
The bind addr should default to "0.0.0.0", so it's not a required parameter. What fails if you take these out (seems like it might be a regression)?